java/client/src/main/java/glide/api/commands/PubSubClusterCommands.java

```diff
@@ -8,21 +8,22 @@
  *
  * @see <a href="https://redis.io/docs/latest/commands/publish/">Pub/Sub Commands</a>
  */
-public interface PubSubClusterCommands {
+public interface PubSubClusterCommands extends PubSubBaseCommands {
```

Comment:
We don't extend interfaces to avoid multiple inheritance, since `*BaseCommands` implemented in BaseClient

Reply:
Is this something were concerned about? It looks like this only really matters when you have default methods or constants in the interfaces that conflict? If it's only abstract methods it doesn't seem to make a difference.

I could just copy the definition of publish() into PubSubClusterCommands, but that's just another place to update the documentation. It also sort of obscures the relationship between PubSubCommand implementations and PubSubClusterCommand implementations.

Reply:
Actually, don't need to duplicate the definition of the non-boolean overload. Done.